### ISSUE TITLE
Figure of what to do with Jest and ES Modules

### ISSUE THREAD

**Issue by kr8n3r** [alphagov/paas-admin#2483]
## What

**TLDR; We need to find a way to make ES module dependencies work with Jest and TS-jest without affecting our javascript bundle meant for the client-side browser.**

We have an increasing [number of dependencies](https://github.com/alphagov/paas-admin/labels/type%3A%20module) moving to [ES Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
They link to [Sindre Sorhus's write-up](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)
 
 Tests now break in Jest test with 

```
Jest encountered an unexpected token

 Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

By default "node_modules" folder is ignored by transformers.

 Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

You'll find more details and examples of these config options in the docs:
 - https://jestjs.io/docs/configuration
 - For information about custom transformations, see:
 - https://jestjs.io/docs/code-transformation
```

For use with Typescript use use a jest wrapper, `ts-jest`and they have [docs on ES module support](https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/)

In most cases we already use `import` instead of `require`.

## Why 

So that we can update dependencies that have move to ES modules

## It’s done when 

* tests pass and update pull requests are merged

## If we don’t do this 

We'll be stuck with out of date dependencies.
